api/v1beta1/events.go

```diff
@@ -258,19 +258,19 @@ var Events = map[EventReason]Event{
 	EventDisruptionCronCreated: {
 		Type:                        corev1.EventTypeNormal,
 		Reason:                      EventDisruptionCronCreated,
-		OnDisruptionTemplateMessage: "DisruptionCron created",
+		OnDisruptionTemplateMessage: "DisruptionCron created with hash %s",
```

Comment:
I made this change for the `EventDisruptionCronUpdated` because a user can edit multiple times a disruption cron.  The problem is the agregatorre only allow the first emitted event. By customizing the message it consider the event as a new one and don't drop it. For what I know of disruption a same event is not sent twice time because a disruption is immutable. Also for disruption it's working in this state and I want to modify it the less as I can to reduce the risk of regression.